src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/impl/TangledCommitDetectionImpl.java

```diff
@@ -0,0 +1,84 @@
+package de.uhd.ifi.se.decision.management.jira.extraction.impl;
+
+import java.util.List;
+
+import de.uhd.ifi.se.decision.management.jira.extraction.ChangedFile;
+import de.uhd.ifi.se.decision.management.jira.extraction.Diff;
+import de.uhd.ifi.se.decision.management.jira.extraction.TangledCommitDetection;
+
+public class TangledCommitDetectionImpl implements TangledCommitDetection {
+
+	@Override
+	public void calculatePredication(Diff diff) {
+		this.calculatePackageDistances(diff);
+		this.standardization(diff);
+	}
+
+	@Override
+	public void calculatePackageDistances(Diff diff) {
+		int numberOfFiles = diff.getChangedFiles().size();
```

Comment:
@lalilalalalu , please split this method into smaller private sub methods. Thanks.